src/quo/components/list/item.cljs

```diff
@@ -43,13 +43,13 @@
   [size]
   (case size
     :small 36
-    40))
+    (min size 40)))
 
 (defn size->container-size
   [size]
   (case size
     :small 52
-    64))
+    (min size 64)))
```

Comment:
this seems like it's wrong to me, why did you make these changes? 🤔

Reply:
Changed. Needed to changed to smaller size to match the design.